Xamarin.Forms.Platform.WinRT/CellControl.cs

```diff
@@ -235,6 +235,7 @@ void SetCell(object newContext)
 						defaultContext = listViewController.GetDisplayTextFromGroup(newContext);
 
 					cell = listViewController.CreateDefaultCell(defaultContext);
+					cell.BindingContext = defaultContext;
```

Comment:
I checked with myself, and we both agree, that's how it should be. Let's imagine a user is doing this:
```
<DataTemplate>
  <ViewCell BindingContext="{StaticResource Foo}">
    ...
  </ViewCell>
</DataTemplate>
```
Using `cell.BindingContext = defaultContext;` will override a user-defined BindingContext, and `SetBindingContext()` won't.
Now, this is very minor, as I don't know any user with a sane mind that would manually set the BindingContext to a TextCell in a DataTemplate, as all of the cells will look exactly the same. But we have to be inclusive, and XF should be useful for the users with sane minds, and... the others too :Smh